common/test/acceptance/tests/studio/test_studio_course_info.py

```diff
@@ -0,0 +1,145 @@
+    """
+
+    def _create_and_verify_update(self, message):
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update(message)
+        self.assertTrue(self.course_updates_page.is_first_update_message(message))
+
+    def setUp(self, is_staff=False, test_xss=True):
+        super(UsersCanAddUpdatesTest, self).setUp()
+        self.auth_page = AutoAuthPage(self.browser, staff=True)
+        self.dashboard_page = DashboardPage(self.browser)
+        self.course_updates_page = CourseUpdatesPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+
+    def test_course_updates_page_exists(self):
+        """
+        Scenario: User can access Course Updates Page
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            When I visit the page
+            Then I should see any course updates
+            And I should see the new update button
+        """
+        self.course_updates_page.visit()
+        self.course_updates_page.wait_for_page()
+        self.assertTrue(self.course_updates_page.are_course_updates_on_page())
+        self.assertTrue(self.course_updates_page.is_new_update_button_present)
+
+    def test_new_course_update_is_present(self):
+        """
+          Scenario: Users can add updates
+              Given I have opened a new course in Studio
+              And I go to the course updates page
+              When I add a new update with the text "Hello"
+              Then I should see the update "Hello"
+              And I see a "saving" notification
+        """
+        self.course_updates_page.visit()
+        self.assertTrue(self.course_updates_page.is_new_update_button_present())
+        self.course_updates_page.click_new_update_button()
+        self.course_updates_page.submit_update('Hello')
+        self.assertTrue(self.course_updates_page.is_first_update_message('Hello'))
+
+    def test_new_course_update_can_be_edited(self):
+        """
+        Scenario: Users can edit updates
+            Given I have opened a new course in Studio
+            And I go to the course updates page
+            When I add a new update with the text "Hello"
+            And I modify the text to "Goodbye"
+            Then I should see the update "Goodbye"
+        """
+        self._create_and_verify_update('Hello')
+        self.assertTrue(self.course_updates_page.is_edit_button_present())
+        self.course_updates_page.click_edit_update_button()
+        self.course_updates_page.submit_update('Goodbye')
+        self.assertFalse(self.course_updates_page.is_first_update_message('Hello'))
+        self.assertTrue(self.course_updates_page.is_first_update_message('Goodbye'))
```

Comment:
Do you need to check for both Hello and Goodbye? If one is present the other won't be.